tableauserverclient/_version.py

```diff
@@ -120,7 +120,7 @@ def versions_from_parentdir(parentdir_prefix, root, verbose):
         dirname = os.path.basename(root)
         if dirname.startswith(parentdir_prefix):
             return {
-                "version": dirname[len(parentdir_prefix) :],
+                "version": dirname[len(parentdir_prefix):],
```

Comment:
were these changes made automatically by pycharm?

Reply:
Those were pycodestyle errors that I fixed manually.

Reply:
great thanks for fixing those!